l10n_it_fatturapa_out/data/invoice_it_template.xml

```diff
@@ -213,7 +213,7 @@ e 'line' per riga di fattura (a seconda del livello in cui sono chiamati)
                     <TipoDato>Valuta</TipoDato>
                     <RiferimentoTesto t-esc="line.currency_id.name" />
                     <RiferimentoNumero
-                        t-esc="format_price(line, original_currency=True)"
+                        t-esc="format_numbers_two(get_importo_totale(record))"
```

Comment:
`get_importo_totale` è il totale fattura, non il totale di riga.

Reply:
Ah giusto in realtà volevo mettere `get_importo` ma mi sono accorto non andasse bene neanche quello

La soluzione attuale potrebbe funzionare?